addons/base/views.py

```diff
@@ -151,7 +152,7 @@ def get_addon_user_config(**kwargs):
     'movefrom': 'write',
 }
 
-def check_access(node, auth, action, cas_resp):
+def check_access(target, auth, action, cas_resp):
```

Comment:
Many of these changes just refactor `node` to `target`, it's pretty hard to read without this change.